src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,131 @@
+    private final User shortPasswordUser = new User();
+    private final User shortLoginUser = new User();
+    private final User misMathcPatternUser = new User();
+
+    @BeforeEach
+    void setUp() {
+        Storage storage = new Storage();
+        storage.people.clear();
+        defaultUser.setAge(18);
+        defaultUser.setLogin(DEFAULT_LOGIN);
+        defaultUser.setPassword(DEFAULT_PASSWORD);
+
+        defaultCopyUser.setAge(DEFAULT_AGE);
+        defaultCopyUser.setLogin(DEFAULT_LOGIN);
+        defaultCopyUser.setPassword(DEFAULT_PASSWORD);
+
+        nullUser.setAge(null);
+        nullUser.setLogin(null);
+        nullUser.setPassword(null);
+
+        nullLoginUser.setPassword(DEFAULT_PASSWORD);
+        nullLoginUser.setLogin(null);
+        nullLoginUser.setAge(DEFAULT_AGE);
+
+        nullPasswordUser.setAge(DEFAULT_AGE);
+        nullPasswordUser.setLogin(DEFAULT_LOGIN);
+        nullPasswordUser.setPassword(null);
+
+        nullAgeUser.setLogin(DEFAULT_LOGIN);
+        nullAgeUser.setPassword(DEFAULT_PASSWORD);
+
+        shortLoginUser.setLogin(SHORT_LOGIN);
+        shortLoginUser.setPassword(DEFAULT_PASSWORD);
+        shortLoginUser.setAge(DEFAULT_AGE);
+
+        shortPasswordUser.setLogin(DEFAULT_LOGIN);
+        shortPasswordUser.setPassword(SHORT_PASSWORD);
+        shortPasswordUser.setAge(DEFAULT_AGE);
+
+        misMathcPatternUser.setPassword(MISMATCH_PASSWORD);
+        misMathcPatternUser.setLogin(MISMATCH_LOGIN);
+        misMathcPatternUser.setAge(DEFAULT_AGE);
+    }
+
+    @Test
+    void checkIfUserIsRegistered_Ok() {
+        registrationService.register(defaultUser);
+        assertEquals(defaultUser, storageDao.get(defaultUser.getLogin()));
+    }
+
+    @Test
+    void checkCurrentLoginExistException_Ok() {
+        registrationService.register(defaultUser);
+        assertThrows(CurrentLoginIsExistsException.class, () ->
+                        registrationService.register(defaultCopyUser),
+                "CurrentLoginIsExist must thrown if current login is located in storage.");
+    }
+
+    @Test
+    void checkInvalidInputDataIfUserNull_Ok() {
+        assertThrows(InvalidInputDataException.class, () ->
+                        registrationService.register(nullUser),
+                "InvalidInputData must thrown if user is null or empty.");
+    }
+
+    @Test
+    void checkInvalidInputDataExceptionIfLoginNullOrEmpty_Ok() {
+        assertThrows(InvalidInputDataException.class, () ->
+                        registrationService.register(nullLoginUser),
+                "InvalidInputData must thrown if Login is null or empty.");
+    }
+
+    @Test
+    void checkInvalidInputDataIfPasswordNullOrEmpty_Ok() {
+        assertThrows(InvalidInputDataException.class, () ->
+                        registrationService.register(nullPasswordUser),
+                "InvalidInputData must thrown if Password is null or empty.");
+    }
+
+    @Test
+    void checkShortPasswordException_Ok() {
```

Comment:
why the all tests named as ok? it should be only for cases that successfully creates the user. 
the good examples of the method names:
- register_nullPassword_notOk() 
- register_shortPassword_notOk() 
- register_loginIsEmpty_notOk()

Check and fix the all method names please